apps/amakrushi/src/components/PhoneView/ChatWindow/Comps.tsx

```diff
@@ -65,20 +65,21 @@ export const RenderComp: FC<any> = ({
                 if (isDisabled) {
                   toast.error('Cannot answer again');
                 } else {
-                  handleSend('text', choice.key);
```

Comment:
there needs to be a condition to set the messages to [] because ,this should happen only when we select from the example options.